primitives/chain-rococo/src/lib.rs

```diff
@@ -42,8 +43,26 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 };
 
 #[derive(parity_scale_codec::Encode, parity_scale_codec::Decode, Debug, PartialEq, Eq, Clone)]
+#[allow(non_camel_case_types)]
 pub enum Call {
-	MockPallet,
+	/// Westend bridge pallet.
+	///
+	/// Note this index has to be kept in-sync with Rococo runtime configuration:
+	/// https://github.com/paritytech/polkadot/blob/master/runtime/rococo/src/lib.rs#L272
+	#[codec(index = 40)]
+	BridgeGrandpaWestend(BridgeGrandpaWestendCall),
+}
+
+#[derive(parity_scale_codec::Encode, parity_scale_codec::Decode, Debug, PartialEq, Eq, Clone)]
+#[allow(non_camel_case_types)]
+pub enum BridgeGrandpaWestendCall {
+	#[codec(index = 0)]
```

Comment:
We should have a similar comment here. For example, what if somebody flips the extrinsic declarations in the pallet and doesn't update this?

Reply:
Added a test for this.